test/runtests.jl

```diff
@@ -47,7 +47,7 @@ end
 obj(model, model.meta.x0)
 @assert neval_obj(model) == 1
 
-reset!(model)
+NLPModels.reset!(model)
```

Comment:
This probably means we should have `import LinearOperators.reset!` before defining `reset!` in `NLPModels`?!